WordPress/src/main/java/org/wordpress/android/ui/accounts/UnifiedLoginTracker.kt

```diff
@@ -0,0 +1,64 @@
+package org.wordpress.android.ui.accounts
+
+import org.wordpress.android.BuildConfig
+import org.wordpress.android.analytics.AnalyticsTracker.Stat.UNIFIED_LOGIN_STEP
+import org.wordpress.android.ui.accounts.UnifiedLoginTracker.Source.DEFAULT
+import org.wordpress.android.util.analytics.AnalyticsTrackerWrapper
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class UnifiedLoginTracker
+@Inject constructor(private val analyticsTracker: AnalyticsTrackerWrapper) {
+    private var currentSource: Source = DEFAULT
+    private var currentFlow: Flow? = null
+    fun track(step: Step) {
+        track(step = step)
+    }
```

Comment:
This puts the app in a infinite loop when called, eventually crashing it. Maybe we should just use the other `track` function and invert the parameters order?

Reply:
ah damn, I've added the named nullable parameters as one of the steps to clean it up but I haven't noticed this obvious non-sense. Thanks for pointing it out! I'll just pass the `currentFlow` as a parameter instead.